### ISSUE TITLE
Unexpected argument y in explain predictions

### ISSUE THREAD

**Issue by wisespira** [alteryx/evalml#3514]
![image](https://user-images.githubusercontent.com/41285415/167909349-f156820a-80da-498c-a9b8-8d32b99cfc9e.png)

**Comment by freddyaboulton** [comment]
Thanks for filing @wisespira ! If you have a target variable, I think you're looking for `explain_predictions_best_worst`

https://evalml.alteryx.com/en/stable/autoapi/evalml/model_understanding/index.html#evalml.model_understanding.explain_predictions_best_worst

Let us know if that fixes the issue!

**Comment by wisespira** [comment]
Thanks for the super quick response @freddyaboulton !
I tried explain_predictions_best_worst, I fed it my target column/series and it seems upset about "cross_entropy encountered ValueError with message (y contains previously unseen labels: 'Yes'"
Can I only input numeric 1/0 or is it only the true values I need to pass?
  
![image](https://user-images.githubusercontent.com/41285415/167949317-55232974-30f3-4360-8f49-ad46561a1f6f.png)

![image](https://user-images.githubusercontent.com/41285415/167949709-78a1cff9-939c-4135-b3dc-5abcde961374.png)

**Comment by freddyaboulton** [comment]
@wisespira Absolutely! Happy to help.

For context, `prediction_explanations_best_worst` scores your pipeline on the data you pass it in order to determine the "best" and "worst" data points to explain. For classification, that metric is `cross_entropy` and for `cross_entropy` to work it needs to convert the string values in your pipeline to numbers. The error you're seeing is because the `Yes` label was not encountered during pipeline training, so the pipeline doesn't know how to encode it. 

 I'm not 100% sure why `Yes` would not be in the training data. As a workaround, I can suggest going back to `explain_predictions`  (😂 ) and picking the row indices you're interested in explaining. This is an example of how you would use it, which I got from the user guide [here](https://evalml.alteryx.com/en/stable/user_guide/model_understanding.html#Explaining-Predictions)

```python
from evalml.model_understanding.prediction_explanations import explain_predictions

table = explain_predictions(pipeline=pipeline_binary, input_features=X_holdout, y=None, indices_to_explain=[3],
                           top_k_features=6, include_explainer_values=True)
print(table)
```

Hope this helps! Let me know either way.

**Comment by wisespira** [comment]
I tried prediction_explanations_best_worst and it worked perfectly ! explain_predictions could identify indices_to_explain or y.

I found that Alteryx python tool has downloaded  __version__ = '0.13.2' which appears to be why I don't have the extra parameters.
![image](https://user-images.githubusercontent.com/41285415/168041780-16346aaa-c3a3-4a99-9315-132b4cace1ee.png)

@freddyaboulton Any idea why Alteryx would download such an old version? If no idea, no worries :D you've been really helpful ! Can close off this issue.
